WPFWijkagent/WPFWijkagent/OffenceController.cs

```diff
@@ -46,11 +46,13 @@ public void SetOffenceData(string description, OffenceCategories category, DateT
         /// <param name="location">offence location</param>
         public void SetOffenceData(string description, OffenceCategories category, DateTime dateTime, Location location)
         {
-            Offence NewOffence = new Offence();
-            NewOffence.Category = category;
-            NewOffence.Description = description;
-            NewOffence.DateTime = dateTime;
-            NewOffence.LocationID = location;
+            Offence NewOffence = new Offence
```

Comment:
voegd dit niet twee overbodige regels toe ;p?

Reply:
In principe wel maar dit ziet er (vind ik) leesbaarder uit ;)